1-mandatory/3-slideshow/slideshow.js

```diff
@@ -1 +1,53 @@
+  "https://lh3.googleusercontent.com/TFN2XckFjZ5Sy93PiLg0MTKCM1oaoxbhICEvSKCxmTCPevr2znN1Oz6hpbbhVc5e4a2suT7U_uL43d8tUsOB52XmEoFRbiXp9xK_FEAgFzwDnYjXhfvg-IsXQ3VBlDBZ7foJgXz8JOcNR0Im7HZlnzM1NWzPAB6i9ve-ojM4AaSD4R3S6ltR1JEz4uBnQ1a8ns3MrbsohphdUfVAk6j7c5bvsfiu_h50MIBDVboTEewEiGYx-2PgN1jPX38mcfePmbgTWyHV_5DkRbe_UUIpG7Q-m1o0Fhmjx7kNXhf06kPD0OoJpW7hmOXCVtrhk3Tpk-bzN14l0l5x0MW0C9IeTYde3-93glbumG7ppgWLnIvDrrtADx6HPOs49AKXqLrzMV4h0wYxBHLSOgidpOny21pSqzAutkiPWYdejyQyZ_QSVBW0_2YiZwk4TkRFrUcRDnT4XggM6s8f4CgYrezJ0uXr3_0r0WQV3EHakuc3XKys7uDQIBak89QMOYEEmdmiutS6vkcvqr-yDpMp8KZ2vHXHoQ3Zdpvmy5BKVTNTtfTRIbEoO17EBMYNCHEbM3e6UB8kbozppzqLE66zqPqpMMYSWaD1Upy3RMvv3MHVFT1BxRoYOa8EQHltgg6Bt08hoRxSF2jzjxf6ohQff4LK-aG_J_Z3IhoZF_j0ko2xlcDKhPE9Np6eKMiMkEPzpSzR0skUOgv0ypnXFVQL5cJ7foUxyp-1SozUOvoVIJV_rJ3daJbUs9d-gOtmKzF-RQ=w711-h948-no?authuser=0",
+  "https://lh3.googleusercontent.com/bXwFEiXWxh7xkhGWBPNvAAjcQU_AiPUb20Ot1OuWfGE4HoUgOzUZR1tCLrady7FdEj5xiRL0H32nzJ0t4ojBihXLjNOtBmzYLwFbjZ-RS_fw03fkfw-I8PsfsZkPPf0UxMtSqJyXrqIn8rkhvJ24uR3GPoafCiBhOmrK0vcw8GJh8AC7rc74ZuAkR_DFS2dI8i_Er_ukr7-I5D4d_f0nhkgmh3jCkGUjOBoDV07ASYdG90Z65gl4v2tbucFbHSaRa6u0HGL5cSBzqbrivGVxob6HnEM8Sn7zFE9Wbh_HrKqqZfZIVwTP_stk89MKZOMPEuzFqMcXuYLjZvj3NaiUUgQkWm4rPDVLTOOKtSqnCAD2WXG4VEgcZqEdX1nu5avRfN3cAOqyWmAmf4uZpIqQ6xhtmz4NhZE1_o4BnvZQWkAiO84FRqtsOgU2EyVtG9sFRxSJPUAPJrY7YhRsUXtbLMTlTc-BJfyM-jdivsfDW5YGHtmpOvdQk24pG88A4uaS-mit42PBp7DU560g75i220nKmweHcjgmFEJkpKVRK9AgLJVViITs_Q9dB_1Lsl9wz_RtMilOba5rp2T0HNqU959KIAJ71T14yULXPmTr5S5APsm7DSjEUBOeoonPTPCEKWDPiFrJiT3COFWfpYurYqP5gYXgzrk_nU-QcJ0AP42_MKvQD6lC8vCkTMowwgmpZNiiGQEwVogQvuap-tBEam7wcjxNhyk0AaxlC354YG5VAnwQkChj3Ql2GS2AjQ=w711-h948-no?authuser=0",
+  "https://lh3.googleusercontent.com/A-26ppznGkhSnITdLtwoyyAuyLdYxl8gvBrQqX-V-yuR_gxc7-jJkyX1Sy7yrDob0k6b7MH8EadANw8gqzeANRN7ZoEwZImxw6Q6T6cVis-YErNP3Rj5FJLCybpvqx7GxsrukJnrUQBYs6Uuf9stYlqA2BUS0_zsKwy-UZKvCLhuXhVDQXeOpd6ndGeSwciAx-llyB8cgrkP0w3AWk1iEwpKGEH0UZGRVE9lLoUdegQpM-qbHpeTnFB7KqSqiwhh1AYzdQQYm82zSHx1qFoe7gy_X0p3C2jbrr4biWvS0P3L49iSLfZoPUfaTn4KVqHujFyIEQ2mQYYaQmdlUOL3_mRZJ6bELFYKy_gjR1LPU5qiFeznW91NnYru87fZ28Swql8eaTgFArv0ki7Dl4KnJcWMy_UczHkes5HieD_-oHfd-XZpCS_KdWILVCQnzloS1NOAsbzztnfwCClYFfT9b7xzNOo36UhhLHLeBx_YE1LOrBFZDXl4c6SnUnDxWc74U4dR4OYsgueJ3arEcmTpMk0OZiIboWskeXRpBYGQ6v8Pv3ojwwI9hJeUyL1V_ksCDHEE-XlILrSZ-uCr64F54oLtdOJ7QNDjeXb_DoDtrgnS9LS4isMSKEuYkLnYbV4lvr0DKxQmLtKJYt6YvSixQsEZ5YtR97RIY4wPPJRucVR3Ytbm8BIk48ZWb8buwWlercWI31XDjYQ15_fyOcnP31KI4ikeU3erqt_N8VHQ2bhX9QJs7CiALVsSej8u-A=w711-h948-no?authuser=0",
+];
+let element_image_1 = document.getElementById("image_1");
+element_image_1.setAttribute("src", images[0]);
+
+document.getElementById("next").addEventListener("click", () => {
+  images.push(images[0]);
+  images.shift();
+  element_image_1.src = images[0];
+});
+
+document.getElementById("previous").addEventListener("click", () => {
+  images.unshift(images[images.length - 1]);
+  images.pop();
+  element_image_1.src = images[0];
+});
+
+let interval;
```

Comment:
Great job defining the `interval` before setting/using it with the event listener 👍